requirements/constraints.txt

```diff
@@ -99,4 +99,4 @@ click<8.0.0
 jsonfield2<3.1.0        # jsonfield2 3.1.0 drops support for python 3.5
 
 # temporary constraint ora update until new deployment ready
-ora2==3.6.11
\ No newline at end of file
+ora2==3.6.11
```

Comment:
Isn't the upgrade version the same as before?

Reply:
yeah weird not sure how this is here... looking

Reply:
this was my editor adding a newline, so it should be ok, I saw ending newlines in other files too